python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -440,6 +440,10 @@ def defectIngest(repo, calib_repo, defectfiles):
     - They will be added to the calib registry, but not linked like the flats and biases
     '''
     log = lsst.log.Log.getLogger('ap.pipe.defectIngest')
+    absRepo = os.path.abspath(repo)
+    defect_tarball = os.path.abspath(defectfiles[0] + '.tar.gz')
+    oldDir = os.path.abspath(os.getcwd())
```

Comment:
Why the name oldDir? This makes it sound like it will be removed later. Maybe add a comment here explaining this directory and it's relation to the absRepo path.

Reply:
"Old" as in "before mucking around with `chdir`." Would "startDir" or "baseDir" be better?

Reply:
Yes, sounds good.